packages/http-client-csharp/.tspd/docs/customization.md

```diff
@@ -1,151 +1,212 @@
-# Customizing the generated code
+## Customizing the generated code
```

Comment:
Was the change to the header levels intentional?    Doesn't seem like there's a `#` member anywhere, maybe this is a child doc of something?

Reply:
Fixed